identity/handler.go

```diff
@@ -205,6 +210,30 @@ type adminCreateIdentity struct {
 	Body AdminCreateIdentityBody
 }
 
+// CredentialsConfig is the struct that is being used as part of the identity credentials.
+type CredentialsConfig struct {
```

Comment:
`CredentialsConfig` is different per credential, it can also be e.g. credentials for WebAuthn, OIDC, etc.

If we move this to here, I would suggest `PasswordCredentialsConfig`. I assume you moved it here because of circular dependencies?